### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by kay0u**
## Problem
- *New version upstream*

## Solution
- *Remove ownCloud migration script*
- *Upgrade to version 20.0.7*
- *Fix upgrade for old installs*

## PR Status
- [ ] Code finished.
- [ ] Tested with Package_check.
- [ ] Fix or enhancement tested.
- [ ] Upgrade from last version tested.
- [ ] Can be reviewed and tested.

## Validation
---
*Minor decision*
- **Upgrade previous version** : 
- [ ] **Code review** : 
- [ ] **Approval (LGTM)** : 
- [ ] **Approval (LGTM)** : 
- **CI succeeded** : 

When the PR is marked as ready to merge, you have to wait for 3 days before really merging it.

### CODE REVIEW

**Comment by kay0u** [comment]
!testme

**Review by ericgaspar — DISMISSED** [review]

**Comment by kay0u** [comment]
!testme

**Review by ericgaspar — APPROVED** [review]

**Comment by kay0u** [comment]
!testme

**Review by ericgaspar — APPROVED** [review]

**Comment by Gredin67** [comment]
hey dear maintainers, any plan to test and integrate #341 in one of the next upgrade? @zamentur @kay0u

**Review by alexAubin — APPROVED** [review]
LGTM

**Comment by alexAubin** [comment]
(+1 for @Gredin67 's comment ... but since that PR is already 24 days old and easy (?) to merge, would rather see it integrated in the next testing)

File: scripts/upgrade [hunk 1]
```diff
@@ -65,6 +65,11 @@ if [ -z "$fpm_usage" ]; then
 	ynh_app_setting_set --app=$app --key=fpm_usage --value=$fpm_usage
 fi
 
+# If phpversion doesn't exist, create it
+if [ -z "$phpversion" ]; then
+    phpversion="$YNH_PHP_VERSION"
+fi
```
Comment (alexAubin) [inline scripts/upgrade:71]: N.B. : somebody encountered the corresponding bug today on the forum ... :/ (upgrade crashing because phpversion not defined)

**Pull request merged**